### DISCUSSION TITLE
Plug 'n' Play configuration/admin feature for turning features on/off

### DISCUSSION THREAD

**Discussion by ZenaMel** [mediacms-io/mediacms#602] [Feature Requests]
The base idea would be to allow either developers (via a global config file, such as `features.yaml` or admin users (via the UI)) to turn features/components on/off.

Examples of what I'd think of as "components":

- comments -> seeing/adding/removing/liking comments
- likes/dislikes
...

Turning a component on/off would result in it not being shown in the frontend.

**Comment by mgogoulos** [comment]
great idea and a much better way than editing files and restarting services (which is what you have to do now)!
